org.civicrm.reqangular/src/common/models/settings.model.js

```diff
@@ -16,15 +16,14 @@ define([
       var promise;
 
       return Model.extend({
-
         /**
-         * Calls the get() method of the Settings API, and returns a
-         * SettingsInstance for each setting.
+         * Returns settings
          *
-         * @param  {Object} params
-         * @return {Promise}
+         * @param {Object} params
+         * @return {Promise} resolved with an array of Settings Instances
          */
         get: function (params) {
+          $log.debug('First call!!!');
```

Comment:
Is that your debugging log left? If yes, please remove it. If it is intended, please amend the caption inside the log to make it look more professional.

Reply:
removed!